packages/content/src/enrichments.ts

```diff
@@ -0,0 +1,443 @@
+import {
+  getItemData,
+  getItemGroups,
+  getUser,
+  IGetUserOptions
+} from "@esri/arcgis-rest-portal";
+import {
+  IHubContent,
+  IHubRequestOptions,
+  IEnrichmentErrorInfo,
+  cloneObject,
+  getProp,
+  getItemHomeUrl,
+  getItemApiUrl,
+  getItemDataUrl,
+  getItemThumbnailUrl,
+  includes
+} from "@esri/hub-common";
+import { getContentMetadata } from "./metadata";
+
+interface IMetadataPaths {
+  updateFrequency: string;
+  metadataUpdateFrequency: string;
+  metadataUpdatedDate: string;
+  reviseDate: string;
+  pubDate: string;
+  createDate: string;
+}
+
+/**
+ * The possible values for updateFrequency
+ *
+ * @enum {string}
+ */
+export enum UpdateFrequency {
+  Continual = "continual",
+  Daily = "daily",
+  Weekly = "weekly",
+  Fortnightly = "fortnightly",
+  Monthly = "monthly",
+  Quarterly = "quarterly",
+  Biannually = "biannually",
+  Annually = "annually",
+  AsNeeded = "as-needed",
+  Irregular = "irregular",
+  NotPlanned = "not-planned",
+  Unknown = "unknown",
+  Semimonthly = "semimonthly"
+}
+
+enum DatePrecision {
+  Year = "year",
+  Month = "month",
+  Day = "day",
+  Time = "time"
+}
+
+function getMetadataPath(identifier: keyof IMetadataPaths) {
+  // NOTE: i have verified that this will work regardless of the "Metadata Style" set on the org
+  const metadataPaths: IMetadataPaths = {
```

Comment:
Could probably just define this as a `const` outside the scope of this `fn` in place of `IMetadataPaths`?